examples/example-tls/README.md

```diff
@@ -43,14 +43,15 @@ You can use the following script to generate self-signed certificates for grpc-j
 mkdir -p /tmp/sslcert
 pushd /tmp/sslcert
 # Changes these CN's to match your hosts in your environment if needed.
+SERVER_CA_CN=localhost-ca # must be different than SERVER_CN
```

Comment:
Do we need the comment "# must be different than SERVER_CN" ?

The comment is the justification for this change but why leave it in the README file?

Reply:
I suppose not.